src/test/resources/definitions.json

```diff
@@ -229,6 +239,15 @@
       "arguments": {
       }
     },
+    {
+      "source": "events",
+      "vhost": "/",
+      "destination": "events.caseProcessor.deactivateUac",
+      "destination_type": "queue",
+      "routing_key": "events.caseProcessor.deactivateUac",
```

Comment:
I know these have already crept in other places, but I don't think the routing key shouldn't include our consuming service. It should just describe the event, e.g. `events.deactivateUac`.

Reply:
I agree, but I think that's tech debt for another ticket - needs to be sorted out consistently. People have lost/forgotten the meaning of routing keys, or conflated it with queue name

Reply:
Fair enough, we do need to go through putting in proper routing keys everywhere so no point enforcing it yet.